spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/route/support/AbstractRibbonCommand.java

```diff
@@ -114,8 +114,16 @@ protected ClientHttpResponse run() throws Exception {
 		final RequestContext context = RequestContext.getCurrentContext();
 
 		RQ request = createRequest();
-		RS response = this.client.executeWithLoadBalancer(request, config);
-
+		RS response;
+		// Note: using isRetriable to determine if spring-retry or netflix is are managing loadbalancing concerns.
+		if (request != null && request.isRetriable()) {
```

Comment:
Can you explain in a bit more detail about why `isRetriable()` says it's spring retry vs not? As far as I know, it's if you are using ribbon `RestClient` or not.

Reply:
I didnt see any clarification on this...

Reply:
Thanks @ryanjbaxter. @pway99?

Reply:
@spencergibb and @ryanjbaxter this is a good question.  There are 3 configurations I have found that affect the restclient configuration.  The first is having a RestTemplate defined and the second is the zuul.retryable property.   

1 Having a RetryTemplate defined a RetryableRibbonLoadBalancingHttpClient is used otherwise its a RibbonLoadBalancingHttpClient.
2 The zuul.retryable property is used as a flag in several locations for defining a request as retryable or not. 
3 The ribbon.httpclient.enabled property enables the HttpClientRibbonConfiguration

looking at these three configurations I would say that the ribbon.httpclient.enabled property most likely defines the RestClient as ribbon or not.

overall its a challenging configuration, I will give some more thought to inline comments.

Thanks again